modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v2/form_fields.rb

```diff
@@ -170,6 +167,9 @@ def issue_decision_date_fields(index)
 
         # rubocop:disable Metrics/MethodLength
         def boxes
+          number_of_issues_on_second_pg = Structure::NUMBER_OF_ISSUES_SECOND_PAGE
```

Comment:
adding these variables to avoid having to break up the blocks below into multiple lines since adding the constant knocks it over the max line length - easier to read this way imo.